logic/block.js

```diff
@@ -444,10 +444,9 @@ Block.prototype.getHash = function (block) {
  * Returns send fees from constants.
  * @param {block} block
  * @return {number} fee
- * @todo delete unused input parameter
  */
-Block.prototype.calculateFee = function (block) {
-	return constants.fees.send;
+Block.prototype.calculateFee = function () {
```

Comment:
remove this function entirely. Not used anywhere.

Reply:
Done!